packages/e2e-test-app/test/__snapshots__/ViewComponentTest.test.ts.snap

```diff
@@ -359,7 +368,7 @@ exports[`ViewTests Views can have background colors 1`] = `
 {
   "AutomationId": "background-color",
   "Background": "#FF527FE4",
-  "BorderBrush": "#00000000",
+  "BorderBrush": "#E4000000",
```

Comment:
I think the fact that we're setting a default border brush in ViewViewManager adds a bunch of these deltas. Not clear whether it's harmful for us to do this or not :|